heartbeat/gcp-pd-move.in

```diff
@@ -255,7 +258,7 @@ def detach_disk(instance, disk_name):
 
   device_name = None
   for disk in response['disks']:
-    if disk_name in disk['source']:
+    if disk_name == re.sub('.*disks/',"",disk['source']):
```

Comment:
It seems like this will break already working installations for users upgrading.

Reply:
As current condition by in which is partial match would make problem that it would wrongly take the different disk in the case that disk-a and disk-aa are attached to the host and configured this RA for disk-a, and I was facing this problem recently.

It would be appreciate if you would advice me to avoid upgrading problem.

Reply:
I'll do some testing over the next couple of days to try and find the optimal way of solving this.

Reply:
I've done some testing, and your patch is good for this situation. Thanks.

Reply:
Thank you very much for your support.